### ISSUE TITLE
Rephrasing for clarity in `creating-a-stack-navigator`

### ISSUE THREAD

**Issue by beoliver** [react-navigation/react-navigation.github.io#614]
https://reactnavigation.org/docs/en/hello-react-navigation.html#creating-a-stack-navigator

I believe that the following sentence is rather hard to parse as a reader

> `createAppContainer` is a function that returns a React component to take as a parameter the React component created by the `createStackNavigator`, and can be directly exported from `App.js` to be used as our App's root component.

perhaps the following would make more sense?

> `createAppContainer` is a function that **takes the React component created by calling `createStackNavigator` and returns a component that** can be directly exported from `App.js` to be used as our App's root component.

**Comment by WoLewicki** [comment]
Thanks for your contribution! It looks good. Could you make a PR with that change?
